package/etc/conf.d/sources/source_syslog/plugin.jinja

```diff
@@ -104,6 +104,7 @@ source s_{{ port_id }} {
             {%- endfor %}
         };
 
+        {% if not ignore_parsing %}
```

Comment:
adding new mode for stream raw message without assigning `sourcetype` and `index`